tests/app/main/test_briefs.py

```diff
@@ -120,6 +129,7 @@ def test_clarification_question_checks_supplier_is_eligible(self, data_api_clien
         self.login()
         data_api_client.get_brief.return_value = api_stubs.brief(status='live')
         data_api_client.get_brief.return_value['briefs']['frameworkName'] = 'Digital Outcomes and Specialists'
+        data_api_client.get_brief.return_value['briefs']['lot'] = 'digital-specialists'
```

Comment:
In terms of `lot` vs `lotSlug` - Yes, good spot, `lotSlug` is newer and preferred.

We had `lot` being passed around a lot, and sometimes it would be the slug and sometimes a (perhaps serialized) `Lot` object.  So we decided to move to `lotSlug` where it's only the slug being passed round and save `lot` for full Lot objects.

But then it turned out that some stuff in the ContentLoader is dependent on `lot` to know which things to load, and we haven't got round to changing it, because it would take a lot of time to fix properly.

So for the time being we have both `lot` and `lotSlug` in the serialization, but yes try to stick to using `lotSlug` in any new code.